docker/monster-auth-req-py/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM python:3-slim
+RUN pip install google-auth requests polling
```

Comment:
This image should take an `ARG` for the versions of each of these libraries, and use the args to pin the `pip install` specs. Otherwise it's not a reproducible build.

Same for the existing image, can you make a ticket for it?